openupgrade_scripts/scripts/website_crm/15.0.2.1/pre-migration.py

```diff
@@ -0,0 +1,63 @@
+from lxml import etree
+from openupgradelib import openupgrade
+
+
+def deprecate_website_crm_privacy_policy(env):
+    if (
+        env["ir.module.module"]
+        .search(["name", "=", "website_crm_privacy_policy"])
+        .state
+        == "installed"
+    ):
+        openupgrade.logged_query(
```

Comment:
Don't do this with SQL, but ORM in this case, and using the recently searched record (+ search the other).